core/modules/admin_bar/css/admin_bar-print.css

```diff
@@ -0,0 +1,6 @@
+/**
+ * Hide the admin bar when printing.
+ */
+#admin-bar {
+  display: none;
+}
```

Comment:
We should also remove the border added to the `body` tag if the admin bar is present. When printing I get a big border at the top of the page where the admin bar would normally be present:

![image](https://user-images.githubusercontent.com/100206/118382987-f2f4d280-b5ae-11eb-813b-421a0a3ccfb4.png)

```suggestion
}
.admin-bar body {
  border-top: none !important;
}

```